### PR TITLE
JIRA: EDGOAIPMH-4 - Define RAML

### PR DESCRIPTION
**Pull request by khandramai**
edge-oai-pmh.raml/OAI-PMH.xsd added
Viachaslau Khandramai <Viachaslau_Khandramai@epam.com>

### CODE REVIEW

**Comment by dcrossleyau** [comment]
I added the api-docs configuration for edge-oai-pmh just now. And tested lint-raml-cop locally.

So the Jenkinsfile can now be modified to enable the "runLintRamlCop" and "publishAPI" jobs.

**Comment by khandramai** [comment]
Please, don't merge this!
This is still in progress.

**Comment by khandramai** [comment]
Hi,
it's ready for review now.
Thanks.

File: ramls/edge-oai-pmh.raml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#%RAML 1.0
+title: 'Edge API - OAI-PMH'
+baseUri: 'https://github.com/folio-org/edge-oai-pmh'
+version: v1
+documentation:
+  -
+    title: 'Edge API - OAI-PMH'
+    content: 'Edge API to interface with FOLIO for 3rd party harvesters to harvest metadta via OAI-PMH'
+types:
+    OAI-PMH: !include OAI-PMH.xsd
```
Comment (epam-pavelk) [inline ramls/edge-oai-pmh.raml]: I would propose to put schema file to a dedicated directory ramls/schemas

File: ramls/edge-oai-pmh.raml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+traits:
+  resps:
```
Comment (epam-pavelk) [inline ramls/edge-oai-pmh.raml]: to improve readability of raml file I would propose to extract the traits to separate files and put them under ramls/traits directory

Comment (craigmcnally) [reply ramls/edge-oai-pmh.raml]: I like this... good suggestion @epam-pavelk

**Comment by craigmcnally** [comment]
Please update the NEWS file

**Pull request merged**